thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -566,6 +572,36 @@ public Response updateAnomalyFunction(@NotNull @PathParam("id") Long id,
   }
 
   /**
+   * Enable apply self-defined alert filter to anomaly function
+   * @param id functionId to be updated
+   * @param alertFilter alert filter in JSON format
+   * @return updated anomaly function
```

Comment:
May let is return only the function Id or 'OK' for a 'PUT' function. You already have the get endpoints if you want to view what you have modified.

Reply:
Agree! Thanks for your suggestion!